rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -119,6 +119,7 @@ separator nonempty Branch "" ;
 -- Match Cases
 CaseImpl. Case ::= Proc13 "=>" Proc3 ;
 separator nonempty Case "" ;
+separator nonempty Case ";" ;
```

Comment:
So a semi-colon is optional? Shouldn't we either make it mandatory or not?

Reply:
Yes, this would make it optional. Making it mandatory would break a lot of code I guess. What are the objections to making it optional?

Reply:
This allows the following:

```
match x0 {
  =x0 => Nil
  =x1 => Nil
  =x2 => Nil;
  =x3 => Nil;
```

which is a bit messy. I'd rather make the pretty-printer not output a `;`, or make it just `separator nonempty Case ";" ;` and fix the rholang we have so far.

Yeah, let's just fix the pretty-printer.